.dockerignore

```diff
@@ -0,0 +1,3 @@
+# The given Dockerfile is for development only and will always
+# clone a clean version from github, so we ignore everything on build.
+**
```

Comment:
Can that potentially interfere with the other docker solution(s)?

Reply:
Will take a look again, but as far as I can see the other solutions also depend on a fresh git clone. Docker context is mainly used in a CI, when the commit automatically triggers the CI, the CI clones the repository at that commit and you can just use that instead of doing a manual clone.